app/helpers/koi/definition_list_helper.rb

```diff
@@ -69,14 +69,13 @@ def definition_tag(&block)
           when ActiveStorage::Attached::One
             tag.dd(attribute_value.attached? ? link_to(attribute_value.filename, url_for(attribute_value)) : "")
           else
-            tag.dd(attribute_value)
+            tag.dd(attribute_value.to_s)
           end
         end
       end
 
       def label_for(**options)
-        options.dig(:label, :text) ||
-          t(attribute, scope: object.model_name.param_key.to_sym, default: attribute.to_s.humanize)
+        options.dig(:label, :text) || object.class.human_attribute_name(attribute)
```

Comment:
@sobakasu are there any examples in Fringe of `t(attribute, scope: object.model_name.param_key.to_sym)` being used? I'm not sure what exactly that would look like but it would be nice not to break it if it's useful (e.g. view helper specific vs requiring active_record).

You could do something like:
```
t(attribute, scope: object.model_name.param_key.to_sym, default: object.class.human_attribute_name(attribute))
```

Reply:
everything in FRG is using en.activerecord.attributes or en.activemodel.attributes apart from RefresherTask which is using that and also the top level attribute names to work around this bug